### DISCUSSION TITLE
使用Java编写的Mirai插件，往群内发送图片报错

### DISCUSSION THREAD

**Discussion by Java0** [mamoe/mirai#1077] [Q&A]
我参考Image类里的注释，用Contact.sendImage(Group,File)，报错NoSuchMethodError。使用ExternalResource.uploadAsImage()，错误依旧

**Comment by Him188** [comment]
这是一个问题, 请临时使用 `Contact.sendImage(Group,File, "png")` 代替

**Reply by Java0** [reply]
好的，问题已解决，感谢回复
